Sources/SwiftDocC/Semantics/MarkupReferenceResolver.swift

```diff
@@ -42,11 +42,11 @@ struct MarkupReferenceResolver: MarkupRewriter {
     // This property offers a customization point for when we need to try
     // resolving links in other contexts than the current one to provide more
     // precise diagnostics.
-    var problemForUnresolvedReference: ((UnresolvedTopicReference, URL?, SourceRange?, Bool) -> Problem?)? = nil
+    var problemForUnresolvedReference: ((UnresolvedTopicReference, URL?, SourceRange?, Bool, String) -> Problem?)? = nil
```

Comment:
nit: should we start labeling these elements so that it's clear what they represent?

Reply:
Agreed. Since this is a function type we can’t have real argument labels but we can still make it clear what each argument means.